diff --git a/src/assets/stylesheets/root-vars.scss b/src/assets/stylesheets/root-vars.scss
index 782afe6c..6443d8ea 100644
--- a/src/assets/stylesheets/root-vars.scss
+++ b/src/assets/stylesheets/root-vars.scss
@@ -88,9 +88,10 @@
   --chat-popup-background-color: #{$darker-grey};
   --chat-popup-text-color: #{$light-text};
   --home-background-color: white;
-  --home-header-background-color: white;
+  --home-header-background-color: #343A40;
   --home-footer-background-color: white;
-  --home-header-link-color: black;
+  --home-header-link-color: white;
+  --home-header-text-color: white;
   --home-footer-text-color: black;
   --home-footer-link-color: #{$dark-grey};
   --home-text-color: black;
diff --git a/src/react-components/home-root.js b/src/react-components/home-root.js
index 6dd5d196..706d96d5 100644
--- a/src/react-components/home-root.js
+++ b/src/react-components/home-root.js
@@ -27,6 +27,7 @@ import styles from "../assets/stylesheets/index.scss";
 import AuthDialog from "./auth-dialog.js";
 import SignInDialog from "./sign-in-dialog.js";
 import MediaTiles from "./media-tiles";
+import ConferenceContent from "../../../src/ui/ConferenceContent.js";
 
 addLocaleData([...en]);
 
@@ -236,46 +237,7 @@ class HomeRoot extends Component {
                 )}
               </div>
             </div>
-            <div className={styles.heroContent} style={{ backgroundImage: configs.image("home_background", true) }}>
-              {!this.props.hideHero &&
-                (this.props.featuredRooms && this.props.featuredRooms.length > 0
-                  ? this.renderFeaturedRoomsHero()
-                  : this.renderNonFeaturedRoomsHero())}
-              {!this.props.hideHero && (
-                <div className={classNames(styles.heroPanel, styles.rightPanel)}>
-                  {showFTUEVideo && (
-                    <div className={styles.heroVideo}>
-                      <video playsInline muted loop autoPlay>
-                        <source src={homeVideoWebM} type="video/webm" />
-                        <source src={homeVideoMp4} type="video/mp4" />
-                      </video>
-                    </div>
-                  )}
-                  <div>
-                    <div className={styles.secondaryLink}>
-                      <a href="/link">
-                        <FormattedMessage id="home.have_code" />
-                      </a>
-                    </div>
-
-                    <IfFeature name="show_discord_bot_link">
-                      <div className={styles.secondaryLink}>
-                        <div>
-                          <FormattedMessage id="home.add_to_discord_1" />
-                        </div>
-                        <img src={discordLogoSmall} />
-                        <a href="/discord">
-                          <FormattedMessage id="home.add_to_discord_2" />
-                        </a>
-                        <div>
-                          <FormattedMessage id="home.add_to_discord_3" />
-                        </div>
-                      </div>
-                    </IfFeature>
-                  </div>
-                </div>
-              )}
-            </div>
+            <ConferenceContent featuredRooms={this.props.featuredRooms} />
             <div className={styles.footerContent}>
               <div className={styles.poweredBy}>
                 <UnlessFeature name="hide_powered_by">
