diff --git a/scripts/deploy.js b/scripts/deploy.js
index 2944adba..a72ee892 100644
--- a/scripts/deploy.js
+++ b/scripts/deploy.js
@@ -5,6 +5,7 @@ import ncp from "ncp";
 import tar from "tar";
 import ora from "ora";
 import FormData from "form-data";
+import path from "path";
 
 if (!existsSync(".ret.credentials")) {
   console.log("Not logged in, so cannot deploy. To log in, run npm run login.");
@@ -100,7 +101,7 @@ const getTs = (() => {
   step.text = "Preparing Deploy.";
 
   step.text = "Packaging Build.";
-  await tar.c({ gzip: true, C: "dist", file: "_build.tar.gz" }, ["."]);
+  tar.c({ sync: true, gzip: true, C: path.join(__dirname, "..", "dist"), file: "_build.tar.gz" }, ["."]);
   step.text = `Uploading Build ${buildEnv.BUILD_VERSION}.`;
 
   let uploadedUrl;
