diff --git a/src/react-components/home-root.js b/src/react-components/home-root.js
index 6dd5d196..6430666c 100644
--- a/src/react-components/home-root.js
+++ b/src/react-components/home-root.js
@@ -28,6 +28,8 @@ import AuthDialog from "./auth-dialog.js";
 import SignInDialog from "./sign-in-dialog.js";
 import MediaTiles from "./media-tiles";
 
+import RoomManagementDialog from "../../../src/ui/RoomManagementDialog";
+
 addLocaleData([...en]);
 
 const isMobile = checkIsMobile();
@@ -209,13 +211,23 @@ class HomeRoot extends Component {
                     </a>
                   </IfFeature>
                   {this.props.showAdmin && (
-                    <a href="/admin" rel="noreferrer noopener">
-                      <i>
-                        <FontAwesomeIcon icon={faCog} />
-                      </i>
-                      &nbsp;
-                      <FormattedMessage id="home.admin" />
-                    </a>
+                    <>
+                      <a href="/admin" rel="noreferrer noopener">
+                        <i>
+                          <FontAwesomeIcon icon={faCog} />
+                        </i>
+                        &nbsp;
+                        <FormattedMessage id="home.admin" />
+                      </a>
+                      <a
+                        href="#"
+                        onClick={e => {
+                          e.preventDefault();
+                          e.stopPropagation();
+                          this.showDialog(RoomManagementDialog);
+                        }}
+                      >
+                        Room Management
+                      </a>
+                    </>
                   )}
                 </div>
               </div>
